tests/integration/test_fim/test_registry/test_registry_file_limit/test_registry_limit_full.py

```diff
@@ -44,8 +44,7 @@
 
 pytest_args:
     - fim_mode:
-        realtime: Enable real-time monitoring on Linux (using the 'inotify' system calls) and Windows systems.
-        whodata: Implies real-time monitoring but adding the 'who-data' information.
+        scheduled:
```

Comment:
Add some description of scheduled mode

Reply:
fixed in commit https://github.com/wazuh/wazuh-qa/pull/2478/commits/dc8c9db733c484bf7e630c5090c8daf470f62229